apps/wpa_supplicant/src/utils/wpa_debug_tinyara.c

```diff
@@ -0,0 +1,762 @@
+/*
+ * wpa_supplicant/hostapd / Debug prints
+ * Copyright (c) 2002-2013, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
+ * See README for more details.
+ */
+
+#include "includes.h"
+
+#include "common.h"
+
+#ifdef CONFIG_DEBUG_SYSLOG
```

Comment:
In my point of view, do not need to include tinyara/config.h because CONFIG_DEBUG_SYSLOG is a config for wpa supplicant not a tizenRT. And this is defined in apps/wpa_supplicant/wpa_supplicant/defconfig